servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpSetCookie.java

```diff
@@ -545,7 +547,7 @@ private static SameSite fromSequence(CharSequence cs, int begin, int end) {
     }
 
     private static boolean equalsIgnoreCaseLower(char c, char k) {
-        return c == k || c >= 'A' && c <= 'Z' && c == k - 32;
+        return CharSequences.equalsIgnoreCaseLower(c, k);
```

Comment:
Consider removing the whole private method and just add a static import for `CharSequences.equalsIgnoreCaseLower`. It has the same name, same params.